pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -1101,16 +1129,16 @@ func Run(ps *pubsub.PubSub) {
 		ctx.subAttestNonce = subAttestNonce
 		//subAttestNonce.Activate()
 
-		pubAttestCert, err := ps.NewPublication(
+		pubEcdhCertConfig, err := ps.NewPublication(
```

Comment:
This is used to publish both the ECDH and attest cert, right? If so it is misnamed.